importApp/src/main/java/com/commercetools/dataimport/orders/OrderCsvLineValue.java

```diff
@@ -0,0 +1,101 @@
+package com.commercetools.dataimport.orders;
+
+
+/**
+ * This class is used as a container for the csv line data
+ */
+
+public class OrderCsvLineValue {
+
+    private String customerEmail;
+    private String orderNumber;
+    private LineItems lineItems;
+    private double totalPrice;
+
```

Comment:
I personally prefer to use just one newline between elements, but we haven't any guidelines on the foramtting/layout of our code. @lauraluiz WDYT?